presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/MetadataConfig.java

```diff
@@ -68,4 +70,18 @@ public MetadataConfig setReassignmentInterval(Duration reassignmentInterval)
         this.reassignmentInterval = reassignmentInterval;
         return this;
     }
+
+    @Min(0)
+    public int getMinimumNodeCount()
+    {
+        return minimumNodeCount;
+    }
+
+    @Config("raptor.minimum-node-count")
+    @ConfigDescription("Minimum number of nodes to activate data recovery and reassignment")
```

Comment:
Minimum nodes required for background data recovery and reassignment procedures to activate